cmd/deployment-check/deployment.go

```diff
@@ -251,6 +264,30 @@ func createDeployment(ctx context.Context, deploymentConfig *v1.Deployment) chan
 	return createChan
 }
 
+// createContainerConfig creates a container resource spec and returns it.
+func createToleration(t string) corev1.Toleration {
+
+	// return blank tolerations if no value provided in yaml file
+	if len(t) == 0 {
+		tolerations := corev1.Toleration{
+			Key:      "",
+			Operator: "Exists",
+			Value:    "",
+			Effect:   "",
```

Comment:
Go ahead and leave the `Key` `Value` and `Effect` fields out if you are going to define them as blank -- they are optional and already [have defaults](https://pkg.go.dev/k8s.io/api/core/v1#Toleration).

Reply:
done